rust/ql/lib/codeql/rust/elements/internal/VariableImpl.qll

```diff
@@ -422,9 +447,13 @@ module Impl {
   }
 
   private import codeql.rust.controlflow.internal.Scope
+  private import codeql.rust.elements.internal.generated.Synth
+
+  private class TVariableAccess = Synth::TPathExpr or Synth::TFormatTemplateVariableAccess;
```

Comment:
This bit is not needed when `VariableAccess` is `abstract`.

Reply:
Ah yes indeed. I initially had that class as non-abstract and later made it abstract.